### DISCUSSION TITLE
Vulkan backend is great

### DISCUSSION THREAD

**Discussion by thewh1teagle** [ggml-org/whisper.cpp#2375] [General]
Just want to say that Vulkan backend makes the project 10x better. 
It works fast as with cuda. it works with Amd/Nvidia/Intel GPUs. Finally almost every computer on Linux/Windows get good performance!
Waiting for the next release!

Some tests I made:
| System                        | Time Taken (1 Hour Audio) | Model          |
|-------------------------------|--------------------------|----------------|
| RTX 3060 on Linux             | 2 minutes                | Medium         |
| AMD Ryzen 5 4500U on Windows  | 25 minutes               | Medium         |
| Mac Mini with M1 and CoreML   | 10 minutes               | Medium         |

**Comment by stopthinking102** [comment]
android supports vulkan . have u tested it on android

**Reply by soundwarrior** [reply]
I also am curious if anyone has tried Vulkan on Android.  With the current CPU implementation in the example, I get about 700 mils on average to decode the average sentence which is pretty good, frankly.  Also, I saw that there is a project called Insanely Fast Whisper, but, of course, it is only in Python currently.  Thanks for any feedback on this.
